lib/creator/transformations/context/__testfixtures__/context-2.input.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+	entry: 'index.js',
+	output: {
+		filename: 'bundle.js'
+	}
+}
```

Comment:
not gonna work. I was planning to integrate it for every transform in v2 for `webpack-cli --add -someRules`